querybook/webapp/components/DataDocQueryCell/DataDocQueryCell.tsx

```diff
@@ -232,6 +228,11 @@ class DataDocQueryCellComponent extends React.PureComponent<IProps, IState> {
         return Object.keys(this.samplingTables).length > 0;
     }
 
+    public get sampleRate() {
+        // -1 or tables don't support sampling, 0 for sample rate is none
```

Comment:
typo `-1 for`

Reply:
// -1 for tables don't support sampling, 0 for default sample rate (which means disable sampling)